graph-kiota/graph-kiota.md

```diff
@@ -0,0 +1,126 @@
+### Graph OpenAPI specification
+
+- OpenAPI v1 specification:https://raw.githubusercontent.com/microsoftgraph/msgraph-metadata/master/openapi/v1.0/openapi.yaml
+- OpenAPI beta specification:https://raw.githubusercontent.com/microsoftgraph/msgraph-metadata/master/openapi/beta/openapi.yaml
+
+### Graph libraries using Kiota
+
+#### Graph Service library:
+
+The Graph Service libraries which contain the request builders will be generated using Kiota
+
+Generation command:
+```
+kiota.exe -d https://raw.githubusercontent.com/microsoftgraph/msgraph-metadata/master/openapi/v1.0/openapi.yaml --language typescript -o path\msgraph-sdk-typescript\src -c GraphServiceClient -n MicrosoftGraph
+```
+- GraphServiceClient generated.
```

Comment:
Can I suggest adding a link to the parameters reference? https://github.com/microsoft/kiota/#parameters-reference